templates/fsharp/app-mvvm/ViewLocator.fs

```diff
@@ -0,0 +1,17 @@
+namespace AvaloniaAppTemplate
+
+open System
+open Avalonia.Controls
+open Avalonia.Controls.Templates
+open AvaloniaAppTemplate.ViewModels
+
+type ViewLocator () =
+    interface IDataTemplate with
+        member __.Match data = data :? ViewModelBase
```

Comment:
Please use one discard ("_") as double is obsolete

Also, I would like to draw your attention to this example:
https://github.com/gsomix/avalonia-fsharp-tutorial/blob/master/ViewLocator.fs